portfolio/src/main/webapp/script.js

```diff
@@ -381,3 +381,35 @@ function postMarker(lat, lng, description) {
 
   fetch('/your-map-data', {method: 'POST', body: params});
 }
+
+google.charts.load('current', {'packages':['timeline']});
+google.charts.setOnLoadCallback(drawChart);
+
+/** Creates a chart and adds it to the page. */
+function drawChart() {
+  let container = document.getElementById('chart-container');
+  let chart = new google.visualization.Timeline(container);
+  let dataTable = new google.visualization.DataTable();
+
+  const rows = [
```

Comment:
sorry I meant more like a global const.
so at the top of the file you might have something like const PROJECT_CHART_ROWS = [...];

Reply:
Oh right, that makes sense. I changed it to a global const and added it at the top of the file.